src/test/java/frc/robot/controllers/DriverJoystickTest.java

```diff
@@ -0,0 +1,112 @@
+package frc.robot.controllers;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+import org.junit.*;
+
+import edu.wpi.first.wpilibj.Joystick;
+import frc.robot.actions.Action;
+import frc.robot.execution.ParallelScheduler;
+import frc.robot.util.Context;
+import frc.robot.util.Trigger;
+
+public class DriverJoystickTest {
+    public DriverJoystick driverJoystick;
+
+    public class Complete extends Action{
+        public void loop()
+        {
```

Comment:
While code style really isn't that big of an issue, the curly braces are very inconsistent. The opening curly braces should all be on the same line as the function declaration. This should not change for if statements, for loops, while loops, or class declarations. It just has to be consistent.